pyproject.toml

```diff
@@ -3,39 +3,39 @@ requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.towncrier]
-    package = "cocotb"
-    directory = "docs/source/newsfragments"
-    filename = "docs/source/release_notes.rst"
-    issue_format = ":pr:`{issue}`"
-    # The first underline is used for the version/date header,
-    # the second underline for the subcategories (like 'Features')
-    underlines = ["=", "-"]
-    all_bullets = false
-
-    [[tool.towncrier.type]]
-        directory = "feature"
-        name = "Features"
-        showcontent = true
-
-    [[tool.towncrier.type]]
-        directory = "bugfix"
-        name = "Bugfixes"
-        showcontent = true
-
-    [[tool.towncrier.type]]
-        directory = "doc"
-        name = "Improved Documentation"
-        showcontent = true
-
-    [[tool.towncrier.type]]
-        directory = "removal"
-        name = "Deprecations and Removals"
-        showcontent = true
-
-    [[tool.towncrier.type]]
-        directory = "change"
-        name = "Changes"
-        showcontent = true
+package = "cocotb"
```

Comment:
IIRC I've proposed removing the unnecessary indentation here before and someone getting mad about it. But that's not a problem if I'm the approver 😁

Reply:
There are two settings, see https://taplo.tamasfe.dev/configuration/formatter-options.html

indent_tables: Indent subtables if they come in order.
indent_entries: Indent entries under tables. 

I activated indent_tables, but not indent_entries. This keeps parts of it, but also indents some of the other subtables. I, sort of, see the point of indenting subtables, but not really the entries. But can be discussed and I can do any setting that is agreed on. (A minor issue is that the VS Code extension seems to use four spaces rather than two, independent of the setting... So I set four spaces now just to have it consistent.)